nexus/tests/output/nexus_tags.txt

```diff
@@ -54,6 +54,10 @@ OPERATION ID                             METHOD   URL PATH
 login_local                              POST     /v1/login/{silo_name}/local
 login_saml                               POST     /login/{silo_name}/saml/{provider_name}
 
+API operations found with tag "metrics"
+OPERATION ID                             METHOD   URL PATH
+silo_metric                              GET      /v1/metrics/{metric_name}
+
```

Comment:
@benjaminleonard @ahl what am I doing here

Reply:
You didn't do this manually right? It's generated 🤖

Reply:
Yeah, but I manually put the tag `metrics` on the endpoint.